src/services/api.js

```diff
@@ -0,0 +1,13 @@
+const base_url = process.env.REACT_APP_BASE_URL;
+export const ApiCallTodos = async (path, method, todo) => {
+  const response = await fetch(base_url + "/todos/" + path, {
+    method,
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: todo ? JSON.stringify(todo) : null,
+  });
+  if (method === "DELETE" || method === "PUT") return response;
```

Comment:
I think we don't need this.

Reply:
As the delete and put not return body with response so response.json() makes error in that case. Thats why i have done it that way